build.sbt

```diff
@@ -2,7 +2,7 @@ name                     := "joern"
 ThisBuild / organization := "io.joern"
 ThisBuild / scalaVersion := "3.4.2"
 
-val cpgVersion = "1.6.13"
+val cpgVersion = "1.6.14+1-df813683"
```

Comment:
Seems this is causing some failures

Reply:
Yup this relies on the other PRs linked above, so this only compiles if you have those published locally. I was hoping that you can review it nonetheless.

Reply:
Right, duh lol.